src/detectors.js

```diff
@@ -36,12 +36,24 @@ exports.getFramework = async function (context) {
 }
 
 exports.getUI = async function (context) {
-  for (const ui of Object.keys(detectors.uis)) {
-    if (await isMatching(detectors.uis[ui].detectors, context)) {
-      return detectors.uis[ui].metas
-    }
+  const uis = new Set()
+
+  await Promise.all(
+    Object.keys(detectors.uis).map(async (ui) => {
+      if (await isMatching(detectors.uis[ui].detectors, context)) {
+        uis.add(detectors.uis[ui])
+      }
+    })
+  )
+
+  if (uis.size === 0) {
+    return null
   }
-  return null
+
+  // lower priority to "css frameworks"
+  const uisArray = Array.from(uis)
+  const ui = uisArray.filter(u => !u.css)[0]
+  return ui ? ui.metas : uisArray[0].metas
```

Comment:
instead of a `css` boolean, perhaps a `priority` value which we set to -1? And then we can sort and return the first value.

Reply:
To be honest I was in doubt to add a similar field instead of `css` field but I should add it to all UI libraries as default value (0 or 1). What do you think?

Reply:
I think priority can be set only in exceptional cases.

Reply:
mmm but `undefined` values go to the bottom using a simple sort

Reply:
@danielroe in order to meet your request I might change the code in this way:

```
return Array.from(uis).map(ui => {
  if (typeof ui.priority === 'undefined') ui.priority = 1
  return ui
}).sort((a, b) => b.priority - a.priority)[0] 
```

Reply:
Or?

```js
return Array.from(uis).sort((a, b) => (b.priority ?? 1) - (a.priority ?? 1))[0] 
```

Reply:
@danielroe yes!